### ISSUE TITLE
The add-in doesn't show after the install instruction has done.

### ISSUE THREAD

**Issue by woodieswu** [gilcos28/HelixGenerator_advanced#1]
The add-in doesn't show after the install instruction has done.
And I found the instruction have no step 6. 
Is anything missing in the instruction?

My action and check:
1. Install python 3.X on your system.
> After typing  "python" in CMD, it enter in python's console.
2. Install the packages numpy and Equation by using the pip install command.
> After typing "pip install numpy" and "pip install Equation" in CMD, it shows "Requirement already satisfied".
3. Unzip the file, rename the directory to "Helix Advanced Generator" and move the directory to fusion's add-in directory.
> C:\Users\woodieswu\AppData\Roaming\Autodesk\Autodesk Fusion 360\API\AddIns\Helix Advanced Generator
4. Fix the path for the installed packages (numpy and Equation) in the code.
> sys.path.append("C:\\Users\\woodieswu\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python39\\site-packages")
7. Enter the path "Tool > Add-in > Scripts and Add-in", select the "Helix Advanced Generator", press the "run" button, and enter into a new sketch, but no “Helix Plus”" in the menu. Go into the path "Tool > Add-in > Scripts and Add-in" again, and find the "Helix Advanced Generator" shows no running icon.
> Full Path : "C:/Users/woodieswu/AppData/Roaming/Autodesk/Autodesk Fusion 360/API/AddIns/Helix Advanced Generator/Helix Advanced Generator.py"

**Comment by gilcos28** [comment]
Hi,
it doesn't miss s step in the instruction, and from my understanding, it should work.
Your path is similar to mine. All looks good.
Since there is no running indication, that the script is running, from your description, I will suspect one of the two:

1. delete all the files from the directories `__pycache__`; this directory also exists in the subdirectories.
2. Your system python path is not set up properly.
3. Downgrade to python 3.7, or basically just installed it next to python 3.9.

Let me know how it goes,
Gil

**Comment by gilcos28** [comment]
Oops, you also need to install numpy.
try this step before:
`pip installed numpy`

Gil

**Comment by woodieswu** [comment]
Hi, thank for your response.
I still stuck on the problem.
"the python path is not set up properly" may be the cause.
I've tried to comment the most of code, and make sure it is executable at first.
Then I uncommented one more row of code and tried to execute it again.
I found it failed to execute when I uncommented the "import numpy as np" in "Helix_Advanced_Command.py".

By the way, I found some information about the fusion 360's python interpreter is located in the other path.
> Full Path : "C:\Users\woodieswu\AppData\Local\Autodesk\webdeploy\production\a86b1e861af1608f41e2e5ec25336d404338c9db\Python\Lib\site-package"

Where I installed numpy and Expression is in the original path.
> Full Path : "C:\Users\woodieswu\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages"

Should I install the numpy and Expression in the first path? Using pip in the cmd seems not work. It just installed the modal in the second path.

**Comment by woodieswu** [comment]
Hi, I solved the problem.
This is caused by an update to windows 2004 under windows 10 environment.
But It still works for the numpy  with the 1.19.3 version.
So I just downgrade the numpy version from 1.19.4 to 1.19.3, anything is fine.
By the way, I found the clue from the exception handler what I added to the "Helix Advanced Generator.py".
And it shows me the link bellow:
[fmod(), after an update to windows 2004, is causing a strange interaction with other code](https://developercommunity.visualstudio.com/content/problem/1207405/fmod-after-an-update-to-windows-2004-is-causing-a.html)
Someone give tips for the version of numpy in the comments.
